reconf/reconf_stress.py

```diff
@@ -72,8 +72,7 @@ def assert_clients(self):
             # Tempesta must be reconfigured in less that 1sec. Errors must not
             # happen after reconfig has finished.
             max_err = rate
-            self.assertTrue((err < max_err),
-                            msg='Client received non 2xx or 3xx responses')
+            self.assertLess(err, max_err)
```

Comment:
The more verbose assertion - the better. Please supply full human-friendly description what the assertion failure means in this particular case. Good message and good test description can give short answer what is wrong without looking though the full test code.

Reply:
assertLess, unfortunaly, don't have description field( And simple assertTrue don't write actual values of err and max_err